src/invidious.cr

```diff
@@ -27,6 +27,7 @@ require "compress/zip"
 require "protodec/utils"
 require "./invidious/helpers/*"
 require "./invidious/*"
+require "./invidious/channels/**"
```

Comment:
As in `spec/helpers_spec.cr`, this should be a single star, to match all files in the `channels` directory, unless you plan to plan to add subfolders, in this case this is the spec file that needs to be fixed.